go.mod

```diff
@@ -8,22 +8,21 @@ require (
 	github.com/hashicorp/hcl v1.0.1-0.20190611123218-cf7d376da96d // indirect
 	github.com/imdario/mergo v0.3.8 // indirect
 	github.com/newrelic/infra-integrations-sdk v1.0.1-0.20201016141521-6fdfe28ad6c0
-	github.com/newrelic/newrelic-telemetry-sdk-go v0.4.0
+	github.com/newrelic/newrelic-telemetry-sdk-go v0.5.1
 	github.com/onsi/ginkgo v1.10.1 // indirect
 	github.com/onsi/gomega v1.7.0 // indirect
 	github.com/pelletier/go-toml v1.2.1-0.20181124002727-27c6b39a135b // indirect
-	github.com/pkg/errors v0.8.1
-	github.com/prometheus/client_golang v0.9.3
-	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
-	github.com/prometheus/common v0.4.0
-	github.com/sirupsen/logrus v1.3.0
+	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.9.0
```

Comment:
Just checked the breaking changes according to [the changelog](https://github.com/prometheus/client_golang/blob/master/CHANGELOG.md#100--2019-06-15) and we don't seem to be using anything there.